src/Kris/LaravelFormBuilder/Fields/FormField.php

```diff
@@ -223,7 +227,7 @@ protected function prepareOptions(array $options = [])
             $this->addErrorClass();
         }
 
-        if ($this->getOption('required') === true) {
+        if ($this->getOption('required') === true || $rulesParser->containsRequired($rules)) {
```

Comment:
I did it this way because the `parse` method returns attributes, and I wasn't sure if that wouldn't be susceptible to change in the future. However, if you decide you do indeed want to use `parse` I'll change the code :-)